app/components/cloud-call.js

```diff
@@ -43,6 +47,7 @@ export default Component.extend({
 
     const init = () => {
       window.addEventListener('resize', resize, false);
+      get(this, 'sanctu').addObserver('cloudsWatch', this, 'getScrollingEl');
       renderer = new THREE.WebGLRenderer({alpha: true});
```

Comment:
Ok, so this is usually way easier to explain in person, buuuuuut:

It's generally considered an anti pattern to use observers in ember, at all. 

The long story is here (it's worth a read on some comp-sci shit):
https://dockyard.com/blog/2015/11/16/best-practices-functional-programming-and-the-observer-effect

The TLDR is that they produce hard to debug errors, and can always be replaced with a more functional code path.

The only exception to the rule is that observers are pretty much the only way to globally watch for route changes (as a changing route is an ember internal that's better not-overridden).  

That said, most apps never need a global route observer either, as those functions can almost always be done in the setupController for each route.

In our case, the main reason we're using a global route observer in the sanctu service is because liquid-fire works well against Bound values, and the mobile slot machine effect uses that animation.

The "better" way to do this is for the sanctu service to be the "single source of truth" as to whether clouds should render.  From there there's a couple ways to respond to that change and either a) short return the render so it doesn't do any work, or b) respond to attrsDidChange (react style when the attr is passed in from the application template.

It's a really important lesson to properly understand - it's bigger than ember, and more about why functional reactive programming and observer models are dangerous architecture @sepowitz - please read this also!